src/models.tsx

```diff
@@ -76,25 +81,20 @@ const Archive = view((get) => {
 
   const content = get(videos)
   if (!content) {
-    return (
-      <>
-        {screen}
-        <ActivityIndicator />
-      </>
-    )
+    return <>{screen}</>
   }
 
   return (
     <>
       {screen}
       <ScrollView contentContainerStyle={{ gap: 0, paddingBottom: $xl }}>
-        <VideoSection title="2023" content={content} />
-        <VideoSection title="2022" content={content} />
-        <VideoSection title="2021" content={content} />
-        <VideoSection title="2020" content={content} />
-        <VideoSection title="2019" content={content} />
-        <VideoSection title="2018" content={content} />
-        <VideoSection title="2017" content={content} />
+        <VideoSection title="2023" content={edition(content, '2023')} />
+        <VideoSection title="2022" content={edition(content, '2022')} />
+        <VideoSection title="2021" content={edition(content, '2021')} />
+        <VideoSection title="2020" content={edition(content, '2020')} />
+        <VideoSection title="2019" content={edition(content, '2019')} />
+        <VideoSection title="2018" content={edition(content, '2018')} />
```

Comment:
this could be done in as single reduce call, but wanted to value readability here over implicitness.